framework/src/types.ts

```diff
@@ -91,7 +91,8 @@ export interface NetworkConfig {
 }
 
 export interface GenesisConfig {
-	epochTime: string;
+	[key: string]: unknown;
```

Comment:
I believe `GenesisConfig` should be strict to given definition, not open to any extra runtime attributes.

Reply:
Module will be instantiated with the genesis config. Other than the required properties from the framework, it will be basically schemaless and each module needs to validate and use it.
Otherwise, module specific config (like # of standby delegates for dpos), will not be able to specify without requiring